.github/workflows/regression-report.yml

```diff
@@ -348,18 +340,19 @@ jobs:
     outputs:
       report_label: ${{ needs.handle_configs_in_pr_description.outputs.report_label }}
     steps:
-      - name: Restore Master Checkstyle Checkout
-        uses: actions/cache@v4
+      - name: Restore cache
+        uses: actions/cache/restore@v4
         with:
           path: .
-          key: "${{ env.CACHE_KEY_MASTER }}"
+          key: ${{ env.CACHE_KEY }}
+          fail-on-cache-miss: true
 
       - name: Restore Generated Config and Project Files
         if: needs.handle_generated_config_bundle.result == 'success'
         uses: actions/cache@v4
         with:
           path: ${{ env.GENERATED_CONFIGS_PATH }}
-          key: ${{ env.CACHE_KEY_GENERATED_CONFIGS }}
+          key: ${{ env.CACHE_KEY }}-generated-configs
```

Comment:
can we avoid extra cache on this ?

Reply:
i don't think so because we need to transfer this files from one job to another. the alternartive is artifacts.

Reply:
just save/generate filles to `.ci-temp/generated-conigs/` folder
and do "cache/store".

Reply:
moved to https://github.com/checkstyle/checkstyle/issues/15635